lib/Common.js

```diff
@@ -187,8 +187,17 @@ Common.prepareAppConf = function(opts, app) {
   else
     env = process.env;
 
-  // Change to double check  (dropped , {pm_cwd: cwd})
-  app.env = [{}, env, app.env || {}].reduce(function(e1, e2){
+  function filterEnv(envObj){
+    let new_env = {};
```

Comment:
In all code use `var` declaration. Why you use `let`?

Reply:
fixed @dipiash